merlin/dag/ops/stat_operator.py

```diff
@@ -27,6 +27,8 @@ class StatOperator(BaseOperator):
     on top of the Operator class.
     """
 
+    fitted = False
```

Comment:
I'm still somewhat concerned that this variable needs to be on the instance level instead of the class level. It works in our test case because we only have one op that needs fitting, but I have a hunch this is going to cause weird errors that are difficult to troubleshoot when we have more than one.

Reply:
Unless we have some code that runs `StatOperator.fitted = True`, changing the class-level attribute we should be ok? If we're concerned about that, maybe can implement an `__init__` method that sets the fitted attribute?